README.md

```diff
@@ -4,42 +4,97 @@
 
 [![Build Status](https://semaphoreci.com/api/v1/duduribeiro/opensanca_jobs/branches/master/badge.svg)](https://semaphoreci.com/duduribeiro/opensanca_jobs)
 
-## README
+## Get Started
 
-### Dependencies
+```
+$ git clone https://github.com/opensanca/opensanca_jobs.git
+$ cd opensanca_jobs
+```
+
+### Environment
+
+Your environment variables goes here in `.env`
+
+```
+# cp .env.sample .env
+```
+
+You can run OpenJobs locally by two ways. 
+- Local classic development.
+- Local Docker development.
+
+### Local Development
+
+#### Dependencies
+
+##### Ruby
 
-#### Ruby Version
 The application uses Ruby 2.5.0. You should install it to run the application. (for development we recommend to use [RVM](https://rvm.io/))
 
-#### Database
+##### Postgres
 
-We use PostgreSQL as database. You should install it to run the app.
+We use PostgresSQL as database. You should install it to run the app.
```

Comment:
It is PostgreSQL and not PostgresSQL

![image](https://user-images.githubusercontent.com/771411/36948794-e2f3ed46-1fbe-11e8-839c-f930c13a8b31.png)

Reply:
done...